config/src/main/java/com/typesafe/config/ConfigException.java

```diff
@@ -310,10 +310,10 @@ public NotResolved(String message) {
      * {@link ConfigException.ValidationFailed} exception thrown from
      * <code>checkValid()</code> includes a list of problems encountered.
      */
-    public static class ValidationProblem {
+    public static class ValidationProblem implements Serializable {
 
         final private String path;
-        final private ConfigOrigin origin;
+        final private transient ConfigOrigin origin;
```

Comment:
Yeah I wasn't sure what to do with that, but looking at your commit here b739f4be, it seems like ConfigOrigin is very intentionally not serializable. Maybe I should follow the lead of `ConfigException`, and add custom `writeObject`/`readObject` methods that call `ConfigImplUtil.writeOrigin` and `ConfigImplUtil.readOrigin`?